apps/studio/components/interfaces/App/FeaturePreview/Branching2Preview.tsx

```diff
@@ -0,0 +1,66 @@
+import Image from 'next/image'
+
+import { InlineLink } from 'components/ui/InlineLink'
+import { BASE_PATH } from 'lib/constants'
+
+export const Branching2Preview = () => {
+  return (
+    <div>
+      <Image
+        src={`${BASE_PATH}/img/previews/branching-preview.png`}
+        width={1296}
+        height={900}
+        alt="api-docs-side-panel-preview"
+        className="rounded border mb-4"
+      />
+      <p className="text-sm text-foreground-light mb-4">
+        Branching 2.0 introduces a new workflow for managing database branches without having to use
+        Git. Create branches, review changes and merge back into production all through the
+        dashboard. Read the below limitations and our{' '}
+        <InlineLink href="https://supabase.com/docs/guides/platform/branching">
+          branching documentation
+        </InlineLink>{' '}
+        before opting in.
+      </p>
+      <div className="my-6">
+        <p className="text-sm text-foreground mb-2 font-medium">Limitations:</p>
+        <ul className="list-disc pl-6 text-sm text-foreground-light space-y-1">
+          <li>
+            Custom roles created through the dashboard will not be captured on branch creation
+          </li>
+          <li>
+            Only <code>public</code> schema changes are supported right now
+          </li>
+          <li>Extensions are not included in the diff process</li>
+          <li>
+            Branches can only be merged to <code>main</code>; merging between preview branches is
+            not supported.
+          </li>
+          <li>
+            If your branch is out of date, you can pull in latest changes from <code>main</code>,
+            but keep in mind that all functions will be overwritten.
+          </li>
+          <li>
+            Deleting functions must be done manually on <code>main</code>.
+          </li>
+          <li>Conflicts must be manually resolved</li>
+          <li>
+            If you have run migrations on <code>main</code>, only migrations are captured in new
+            branches (not the full schema).
```

Comment:
```suggestion
            If you have run migrations on <code>main</code>, new branches will be created from existing migrations instead of a full schema dump.
```

I'd also move this to the top of the list, before custom roles.